meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -70,11 +90,22 @@ public Object update(String key, DataUpdater updater) {
 
   @Override
   public Stat exists(String key) {
-    return null;
-  }
-
-  @Override
-  public Object get(String key) {
+    org.apache.zookeeper.data.Stat zkStats;
```

Comment:
Please mark here as temp impl for test only.

Reply:
yes this will be updated with pr 2327 once it's merged